Hestia/app/src/main/java/hestia/backend/RemoveDeviceTask.java

```diff
@@ -0,0 +1,82 @@
+package hestia.backend;
+
+import android.os.AsyncTask;
+import android.util.Log;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URL;
+
+/**
+ * This class will send a DELETE request to the Server in order to delete
+ * the Device "device".
+ */
+
+public class RemoveDeviceTask extends AsyncTask<Void, Void, Void> {
```

Comment:
If you change the third type to Integer, you can have the task return the response code. This is preferable to just printing it and returning null.